src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml

```diff
@@ -35,10 +35,16 @@
                 </div>
             </div>
 
+            <h2 class="govuk-heading-m">What happens next</h2>
```

Comment:
The AC's state the colour is changing - I don't see a change for this unless it's CDN supplied? and I didn't think this was using a CDN?

Reply:
Upon investigation, the new GDS colour has already been implemented.